querybook/webapp/ui/EditableTextField/EditableTextField.tsx

```diff
@@ -17,11 +17,12 @@ export interface IEditableTextFieldProps {
     className?: string;
     readonly?: boolean;
     placeholder?: string;
+    onEdit?: () => void;
```

Comment:
i think its better if we just have a readonly richtext field with a redirect button. Most of the code in EditableTextField is useless if onEdit is provided

Reply:
I think it's similar as when readonly is true, most of the code is also useless. are you saying we should render a readonly rich text for both of them?